images/aws-scripts/deploy-kubeflow.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This shell script is used to deploy kubeflow by kfctl
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+EKS_CLUSTER_NAME="${EKS_CLUSTER}"
+EKS_NAMESPACE_NAME="${EKS_NAMESPACE}"
+
+# Load kubeconfig
+aws eks update-kubeconfig --name=$EKS_CLUSTER_NAME
+
+cd /tmp
+
+# Fetch v1.1-branch kfctl
+wget https://github.com/PatrickXYS/kfctl/releases/download/test1/kfctl_v1.1.0-2-g08ee6e4_linux.tar.gz -O kfctl.tar.gz
```

Comment:
Will this be changed later? Can you extract these into envs? If you have a follow up PR. I am fine to leave them here for now.

Reply:
We might need to come up with a better plan to add Compatibility matrix later.